open-api/patch/pubspec_intl.dart.patch

```diff
@@ -0,0 +1,9 @@
+@@ -11,7 +11,7 @@
+ dependencies:
+   collection: '^1.17.0'
+   http: '>=0.13.0 <0.14.0'
+-  intl: '^0.18.0'
++  intl: '^0.19.0'
```

Comment:
Maybe we should upgrade to a newer version of the generator instead.

https://github.com/OpenAPITools/openapi-generator/pull/17862